src/component/application/application-edit-component.js

```diff
@@ -61,16 +61,7 @@ class ClientApplications extends PureComponent {
             return <ProgressBar indeterminate />;
         }
         const { application, storeApplicationMetaData } = this.props;
-        const {
-            appName,
-            instances,
-            strategies,
-            seenToggles,
-            url,
-            description,
-            icon = 'apps',
-            color,
-        } = application;
+        const { appName, instances, strategies, seenToggles, url, description, icon = 'apps', color } = application;
```

Comment:
if we add a line break and reformat, it will be on multiple lines (prettiers respects that).

I intentionally reverted the prettier commit on master before making this PR so it got to work on a clean slate

Reply:
Wait, I lied 😭 
Maybe they removed that behavior, it was the case before

Reply:
Some are kept. Maybe just in object literals?

```diff
diff --git i/src/store/input-actions.js w/src/store/input-actions.js
index fb4aa87..c59354d 100644
--- i/src/store/input-actions.js
+++ w/src/store/input-actions.js
@@ -9,12 +9,41 @@ export const actions = {
     MOVE: 'MOVE',
 };
 
-export const createInit = ({ id, value }) => ({ type: actions.INIT, id, value });
-export const createInc = ({ id, key }) => ({ type: actions.INCREMENT_VALUE, id, key });
-export const createSet = ({ id, key, value }) => ({ type: actions.SET_VALUE, id, key, value });
-export const createPush = ({ id, key, value }) => ({ type: actions.LIST_PUSH, id, key, value });
-export const createPop = ({ id, key, index }) => ({ type: actions.LIST_POP, id, key, index });
-export const createMove = ({ id, key, index, toIndex }) => ({ type: actions.MOVE, id, key, index, toIndex });
+export const createInit = ({ id, value }) => ({
+    type: actions.INIT,
+    id,
+    value,
+});
+export const createInc = ({ id, key }) => ({
+    type: actions.INCREMENT_VALUE,
+    id,
+    key,
+});
+export const createSet = ({ id, key, value }) => ({
+    type: actions.SET_VALUE,
+    id,
+    key,
+    value,
+});
+export const createPush = ({ id, key, value }) => ({
+    type: actions.LIST_PUSH,
+    id,
+    key,
+    value,
+});
+export const createPop = ({ id, key, index }) => ({
+    type: actions.LIST_POP,
+    id,
+    key,
+    index,
+});
+export const createMove = ({ id, key, index, toIndex }) => ({
+    type: actions.MOVE,
+    id,
+    key,
+    index,
+    toIndex,
+});
 export const createUp = ({ id, key, index, newValue, merge }) => ({
     type: actions.LIST_UP,
     id,
```

Reply:
I have no idea, you know prettier much better than me :1st_place_medal:

Reply:
I think the current change-set looks good as it is now.

Reply:
It is. I find it weird, I'll open an issue

https://prettier.io/playground/#%7B%22content%22%3A%22const%20someObject%20%3D%20%7B%20one%2C%20two%2C%20three%2C%20four%2C%20five%2C%20six%2C%20seven%20%7D%3B%5Cnconst%20%7B%20one%2C%20two%2C%20three%2C%20four%2C%20five%2C%20six%2C%20seven%20%7D%20%3D%20someObject%3B%5Cn%5Cnconst%20someObject%20%3D%20%7B%5Cn%20%20one%2C%5Cn%20%20two%2C%5Cn%20%20three%2C%5Cn%20%20four%2C%5Cn%20%20five%2C%5Cn%20%20six%2C%5Cn%20%20seven%2C%5Cn%7D%3B%5Cnconst%20%7B%5Cn%20%20one%2C%5Cn%20%20two%2C%5Cn%20%20three%2C%5Cn%20%20four%2C%5Cn%20%20five%2C%5Cn%20%20six%2C%5Cn%20%20seven%5Cn%7D%20%3D%20someObject%3B%5Cn%22%2C%22options%22%3A%7B%22printWidth%22%3A80%2C%22tabWidth%22%3A2%2C%22singleQuote%22%3Afalse%2C%22trailingComma%22%3A%22none%22%2C%22bracketSpacing%22%3Atrue%2C%22jsxBracketSameLine%22%3Afalse%2C%22parser%22%3A%22babylon%22%2C%22semi%22%3Atrue%2C%22useTabs%22%3Afalse%2C%22doc%22%3Afalse%2C%22ast%22%3Afalse%7D%7D